### DISCUSSION TITLE
有大佬知道mtk Filogic 8x0 无线加速如何在编译的时候默认启用

### DISCUSSION THREAD

**Discussion by jwkinko** [coolsnowwolf/lede#12177] [Q&A / 快速问答]
Turbo ACC 网络加速设置
里的MTK 无线硬件加速，开启后需要重启路由器，如何默认启用不重启。

**Comment by xiaouex** [comment]
我也是这个芯片的，但是打开无线加速会死机，你编译的时候有没有额外勾选什么还是只需要在luci/application勾选turboacc就行？

**Comment by jwkinko** [comment]
开启无线加速需要重启，我都是默认配置没多选别的。

获取Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
